content/docs/getting_started/quickstart/manual_demo.md

```diff
@@ -391,16 +390,25 @@ The counter in the `bookthief` window will stop incrementing.
 
 ### Configure Traffic Split between Two Services
 
-We will now demonstrate how to balance traffic between two Kubernetes services, commonly known as a traffic split. We will be splitting the traffic directed to the root `bookstore` service between the backends `bookstore` service and `bookstore-v2` service.
+We will now demonstrate how to balance traffic between two Kubernetes services, commonly known as a traffic split. We will be splitting the traffic directed to the root `bookstore` service between the backend services `bookstore-v1` and `bookstore-v2`.
 
 ### Deploy bookstore v2 application
 
-To demonstrate usage of SMI traffic access and split policies, we will now deploy version v2 of the bookstore application (`bookstore-v2`) - remember that if you are using openshift, you must add the security context constraint to the bookstore-v2 service account as specified in the [installation guide](/docs/install/#openshift).
+To demonstrate usage of SMI traffic access and split policies, we will now deploy version v2 of the bookstore application (`bookstore-v2`) - remember that if you are using openshift, you must add the security context constraint to the bookstore-v2 service account as specified in the [installation guide](/docs/install/#openshift). A `bookstore` service will also be created to serve as the root of our traffic split between `bookstore-v1` and `bookstore-v2`.
```

Comment:
It should be documented here that the applications need to be updated to send requests to a new service FQDN.